packages/connect/src/base/entry.ts

```diff
@@ -380,58 +397,75 @@ export class EntryNodes extends EventEmitter {
     }
   }
 
-  /**
-   * Attempts to connect to a relay node
-   * @param id peerId of the node to dial
-   * @param relay multiaddr to perform the dial
-   * @param timeout when to timeout on unsuccessful dial attempts
-   * @returns a PeerStoreEntry containing the measured latency
-   */
-  private async connectToRelay(
-    id: PeerId,
-    relay: Multiaddr,
+  private async establishNewConnection(
+    destination: PeerId,
+    destinationAddress: Multiaddr,
     timeout: number
-  ): Promise<{ entry: EntryNodeData; conn?: Connection }> {
+  ): Promise<ConnResult | void> {
     const abort = new AbortController()
-    const start = Date.now()
-    let connectionEstablished = false
+    let done = false
 
     setTimeout(() => {
```

Comment:
Please document reason for this abort timeout.

Reply:
changed